index.html

```diff
@@ -1,259 +1,250 @@
+
+    <div id="projectimg1"></div>
+    <div id="projectimg2"></div>
+    <div id="projectimg3"></div>
+    <div id="projectimg4"></div>
+  </section>
+  <section class="about" id="about">
+    <div class="container-about">
+      <h2 class="aboutheader">About</h2>
+      <p class="introabout">
+        Lorem ipsum dolor sit amet,
+        consectetur adipiscing elit,
+        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        Ut,
+      </p>
+      <button type=button class="btn3">Get my CV</button>
+    </div>
+    <div class="container-about2">
+      <ul class="grid-container">
+        <li class="aboutcard">
+          <ul class="grid-item1">
+            <li class="about-title">
+              <h3>Languages</h3>
+            </li>
+            <li>JavaScript (es5/es6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </li>
+        <li class="aboutcard">
+          <ul class="grid-item2">
+            <li class="about-title">
+              <h3>Frameworks</h3>
+            </li>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+            <li>Selenium</li>
+          </ul>
+        </li>
+        <li class="aboutcard">
+          <ul class="grid-item3">
+            <li class="about-title">
+              <h3>Skills</h3>
+            </li>
+            <li>Codekit</li>
+            <li>Codepen</li>
+            <li>Github</li>
+            <li>GitLab</li>
+            <li>terminal</li>
+          </ul>
+        </li>
+        <li class="aboutcard">
+          <ul class="grid-item4">
+            <li><img src="/css/dots.png" width="170"  alt="dots"></li> 
+          </ul>
+      </ul>
+    </div>
+  </section>
+  <form action="https://formspree.io/f/mayaljan" method="post" id="form">
+    <fieldset class="contact">
+      <h2 class="contactheader">Get Started</h2>
+      <p class="introcontact">Lorem ipsum dolor sit amet,
+        consectetur adipiscing elit,
+        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        Ut enim ad minim veniam,
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. .</p>
+      <div class="form-group">
+        <label for="name"> Your mail
+          <input type="text" class="name" id="name" name="name" required maxlength="30" />
+        </label>
+        <label for="email"> Your email
+          <input type="email" class="name" id="email" name="email" required />
+        </label>
+      </div>
+      <label class="message-label" for="message">Message</label>
+      <textarea name="message" class="message" id="message" placeholder="  Enter your message here..." required
+        maxlength="500"></textarea>
+      <input type="submit" class="btn1" value="Start collaboration" />
+    </fieldset>
+  </form>
+  <footer>
+    <div class="resume">
+      <p>Get my CV</p>
+    </div>

```

Comment:
- [ ] The idea of putting this in the footer is to allow users to _**click**_ your _**CV**_ and it would direct them to the site where they can see it or download it for their use. I think it would be nice to wrap this in an `<a>` tag instead of `<p>` tag. 👍